archinstall/lib/user_interaction.py

```diff
@@ -659,21 +769,29 @@ def select_driver(options=AVAILABLE_GFX_DRIVERS):
 				elif b'amd' in line.lower():
 					print(' ** AMD card detected, suggested driver: AMD / ATI **')
 
-		initial_option = generic_select(drivers, input_text="Select your graphics card driver: ")
+		initial_option = generic_select(
+			drivers, input_text="Select your graphics card driver: "
+		)
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. Inconsistent behavior, arguments on the same line.